server/common/comments/patch.ts

```diff
@@ -0,0 +1,28 @@
+import { NextApiRequest, NextApiResponse } from "next";
+
+import models from "db/models";
+
+import { error as LogError } from "services/logging";
+
+export default async function path(req: NextApiRequest, res: NextApiResponse) {
+  try {
+    const { id } = req.query;
+    const { hidden } = req.body;
```

Comment:
hidden can be `false` or `null | undefined` if the second happens, we don't want to change the state, as false is not null which is not undefined